src/loaders/lottie/tvgLottieModel.h

```diff
@@ -334,6 +334,148 @@ struct LottieMarker
     }
 };
 
+
+struct LottieTextFollowPath
+{
+    LottieFloat firstMargin = 0.0f;
+    LottieToggle perpendicular = 1;
+    int8_t maskIdx = -1;
+    LottieMask* mask;
+
+    bool set(LottieMask* mask, float frameNo, float scale, LottieExpressions* exps)
+    {
+        this->mask = mask;
+        Matrix m{1.0f / scale, 0.0f, 0.0f, 0.0f, 1.0f / scale, 0.0f, 0.0f, 0.0f, 1.0f};
+        mask->pathset(frameNo, path, &m, exps);
+
+        if (path.pts.count < 2) return false;
```

Comment:
Please put this exception earlier.

Reply:
two lines above I got the path's values. earlier where? am I missing something?

Reply:
Please remove the exception. invalid mask shouldn't be existed right from the get-go.